cairo/ethereum/cancun/state.cairo

```diff
@@ -178,6 +180,63 @@ func get_storage{poseidon_ptr: PoseidonBuiltin*, state: State}(
             created_accounts=state.value.created_accounts,
         ),
     );
+    return value;
+}
+
+func get_transient_storage{poseidon_ptr: PoseidonBuiltin*}(
+    transient_storage: TransientStorage, address: Address, key: Bytes32
+) -> U256 {
+    alloc_locals;
+    let fp_and_pc = get_fp_and_pc();
+    local __fp__: felt* = fp_and_pc.fp_val;
+
+    // read the MappingAddressTrieBytes32U256
+    let dict_ptr = cast(transient_storage.value._tries.value.dict_ptr, DictAccess*);
+    with poseidon_ptr, dict_ptr {
+        let (trie_ptr) = hashdict_read(1, &address.value);
+    }
+
+    // Update the dict_ptr to point to the new dict
+    let new_dict_ptr = cast(dict_ptr, AddressTrieBytes32U256DictAccess*);
+    let original_mapping = transient_storage.value._tries.value.original_mapping;
+    tempvar _tries = MappingAddressTrieBytes32U256(
+        new MappingAddressTrieBytes32U256Struct(
+            transient_storage.value._tries.value.dict_ptr_start, new_dict_ptr, original_mapping
```

Comment:
```suggestion
    tempvar _tries = MappingAddressTrieBytes32U256(
        new MappingAddressTrieBytes32U256Struct(
            transient_storage.value._tries.value.dict_ptr_start, new_dict_ptr, transient_storage.value._tries.value.original_mapping
```

nit but I'd be of opinion to not "move" this to a variable if it's not just re-binded to the same value